ironfish-cli/src/commands/accounts/import.ts

```diff
@@ -39,19 +40,25 @@ export class ImportCommand extends IronfishCommand {
     if (importPath) {
       const resolved = this.sdk.fileSystem.resolve(importPath)
       data = fs.readFileSync(resolved, 'utf8')
-    } else if (process.stdin) {
-      data = ''
-
-      const onData = (dataIn: string): void => {
-        data += dataIn
-      }
-
-      process.stdin.setEncoding('utf8')
-      process.stdin.on('data', onData)
-      while (!process.stdin.readableEnded) {
-        await PromiseUtils.sleep(100)
-      }
-      process.stdin.off('data', onData)
+    } else {
+      const importAccountName= (await cli.prompt('Enter the account name', {
+        required: true,
+      })) as string
+      const importSpendingKey = (await cli.prompt('Enter the account spending key', {
+        required: true,
+      })) as string
+      const importIncomingViewKey = (await cli.prompt('Enter the account incoming view key', {
+        required: true,
+      })) as string
+      const importOutgoingViewKey = (await cli.prompt('Enter the account outgoing view key', {
+        required: true,
+      })) as string
+      const importPublicAddress = (await cli.prompt('Enter the account public address', {
+        required: true,
+      })) as string
+      data = `{"name": "${importAccountName}", "spendingKey": "${importSpendingKey}",
+      "incomingViewKey": "${importIncomingViewKey}", "outgoingViewKey": "${importOutgoingViewKey}",
+      "publicAddress": "${importPublicAddress}"}`
     }
 
     if (data === null) {
```

Comment:
This looks like it will not allow using STDIN to import an account anymore. Instead of just deleting that functionality and replacing it, can you keep all 3 paths?

 *.Import by path
 * Import by STDIN
 * Import by interactive flow

Reply:
You could check if there's any data, and if there is any, then you can just import that versus perform interactive.

Reply:
Updated the code to use process.stdin.isTTY to detect if stdin input is passed or not. This will now either accept a path, stdin input, or switch to interactive mode if neither of those inputs are present. Updated the import statements for fs and cli-ux. Tested locally and all methods checked out.

Reply:
Looks great!